src/applications/vaos/containers/ExpressCareInfo.jsx

```diff
@@ -1,12 +1,12 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
```

Comment:
Could we call this file ExpressCareInfoPage instead of ExpressCareInfo? That's more consistent with how we've been naming other components that cover a whole page.

Reply:
Done